documentation/local.md

```diff
@@ -1,8 +1,59 @@
-# Local environment
+# Development environment
 
-## Docker compose profiles
+## Requirements
+- Docker and Docker compose
+- Make (BSD and GNU are supported)
+- [Stonehenge](https://github.com/druidfi/stonehenge)
+
+## Installation
+
+- Clone the repository
```

Comment:
Should we add a mention here that Stonehenge needs to be started also for local URLs to work?

Reply:
Good point. Yes we should